tests/console/system_prepare.pm

```diff
@@ -23,6 +24,18 @@ sub run {
 
     ensure_serialdev_permissions;
 
+    # Configure serial consoles for virtio support
+    # poo#18860 Enable console on hvc0 on SLES < 12-SP2
+    # poo#44699 Enable console on hvc1 to fix login issues on ppc64le
+    if (!check_var('VIRTIO_CONSOLE', 0)) {
+        if (is_sle('<12-SP2')) {
+            add_serial_console('hvc0');
+        }
+        elsif (get_var('OFW')) {
+            add_serial_console('hvc1');
+        }
+    }
```

Comment:
How about

```perl
add_serial_console('hvc' . is_sle('<12-SP2') ? 0 : 1) if (get_var('VIRTIO_CONSOLE') && check_var('ARCH', 'ppc64le'));
```

or what should happen in the "else"-branch?

Reply:
@okurz a week ago my `if() { … } else { … }` was 'too complex' and now you write this?